ui/desktop/package.json

```diff
@@ -45,9 +45,9 @@
   },
   "cross-os": {
     "prebuild:desktop": {
-      "darwin": "cp -r dist/ electron-app/ember-dist/",
-      "linux": "cp -r dist/ electron-app/ember-dist/",
-      "win32": "(robocopy dist electron-app\\ember-dist /MIR) ^& exit 0"
+      "darwin": "rm -rf electron-app/ember-dist/ && cp -r dist/ electron-app/ember-dist/",
+      "linux": "rm -rf electron-app/ember-dist/ && cp -r dist/ electron-app/ember-dist/",
+      "win32": "rmdir electron-app/ember-dist/ && (robocopy dist electron-app\\ember-dist /MIR) ^& exit 0"
```

Comment:
Is `&&` valid powershell syntax? I don't think its available in most versions

Reply:
@ZedLi I did not tested windows yet, I am on it. But from top of my mind I think you are right, windows is just `&` not `&&`? I am about to run this on Windows and apply changes if need :wink: